pkg/types/gcp/platform.go

```diff
@@ -9,6 +9,11 @@ type Platform struct {
 	// Region specifies the GCP region where the cluster will be created.
 	Region string `json:"region"`
 
+	// PublicZoneID specifies the ID for the public DNS Zone
```

Comment:
In think we should update the documentation https://github.com/openshift/installer/blob/master/docs/user/gcp/customization.md

Reply:
I've added the field to the customization doc